### PR TITLE
#1899921: add registration link on login dialog

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
[AB#1899921](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1899921): add registration link on login dialog

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SignInWindow.java [hunk 1]
```diff
@@ -105,6 +111,16 @@ private void checkAccountAvailability() {
         });
     }
 
+    protected JPanel createSouthAdditionalPanel() {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SignInWindow.java:114]: south -> bottom?

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SignInWindow.java:114]: When will we call this method? Shall we add the panel into form and make it visible if need?

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/SignInWindow.java:114]: it's inherited from `DialogWrapper`

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**